docs/getstarted.rst

```diff
@@ -74,7 +74,7 @@ Copy the following example into your favourite text editor and save it to a file
     result.println { it.trim() }
 
 
-This script defines two processes. The first splits a string in file chunks containing 6 characters,
+This script defines two processes. The first splits a string into 6-character chucks, writing each one to a file with the prefix ``chunk_``,
```

Comment:
Thanks for submitting this but there's a typo in your typo fixing PR, `chucks`  -> `chunks`  :)

Reply:
Ha! Sorry about that. I was too focused on doing the PR part right that I neglected the substance. Hopefully my most recent commit rectified that.